src/style-auro.scss

```diff
@@ -180,44 +190,48 @@ $color-base-black-opacity-15: rgba(255, 255, 255, 0.15);
       border: 1px solid var(--auro-color-ui-default-on-dark);
       color: var(--auro-color-ui-default-on-dark);
   
-      &:active, &:hover {
-        background-color: $color-base-black-opacity-15;
+      @media (hover: hover) {
```

Comment:
what is this?

Reply:
https://drafts.csswg.org/mediaqueries/#hover determines whether the browser supports the hover state